programs/server/Server.cpp

```diff
@@ -1238,8 +1238,13 @@ try
             global_context->setMacros(std::make_unique<Macros>(*config, "macros", log));
             global_context->setExternalAuthenticatorsConfig(*config);
 
-            global_context->loadOrReloadDictionaries(*config);
-            global_context->loadOrReloadUserDefinedExecutableFunctions(*config);
+            if (global_context->isServerCompletelyStarted())
+            {
```

Comment:
Minor. What if somebody changed dictionaries during load? They will never be updated after the server starts?

Reply:
AFAIU a dictionary will be updated on the next config reload (we reload it periodically) if an xml config was changed